src/plugins/Emojis/styling/EmojiPicker.scss

```
@@ -1,11 +1,389 @@
+      --str-chat__background-core-surface-card
+    );
+    --str-chat__emoji-picker-text-color: var(--str-chat__text-primary);
+    --str-chat__emoji-picker-secondary-text-color: var(--str-chat__text-secondary);
+    --str-chat__emoji-picker-hover-background-color: var(
+      --str-chat__background-utility-hover
+    );
+    --str-chat__emoji-picker-selected-background-color: var(
+      --str-chat__background-utility-selected
+    );
+    --str-chat__emoji-picker-border-color: var(--str-chat__border-core-on-surface);
+    // Gentle hairline for the search/footer dividers (the default on-surface border is
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)